ee/clickhouse/sql/events.py

```diff
@@ -256,7 +256,7 @@
 
 NULL_SQL = """
 -- Creates zero values for all date axis ticks for the given date_from, date_to range
-SELECT toUInt16(0) AS total, {trunc_func}(toDateTime(%(date_to)s) - {interval_func}(number)) AS day_start
+SELECT toUInt16(0) AS total, {trunc_func}(toDateTime(%(date_to)s, %(timezone)s) - {interval_func}(number)) AS day_start
```

Comment:
We need to update the UI to communicate that the timezones are aligned with their local timezones now and not UTC

Reply:
I wonder if we do or whether it'll just add noise. This functionality is behind a feature flag anyway so we can do this when we roll it out.